gulpfile.js

```diff
@@ -39,7 +39,7 @@ function gulp_test() {
     return gulp.src(tests)
     .pipe(mocha({
         log: true,
-        timeout: 60000,
+        timeout: 1200000,
```

Comment:
I'l put a comment on the line indicating the exact time since when units are missing our brains do funny things with too many zeros.